packages/web-components/src/components/ai-label/ai-label.ts

```diff
@@ -74,17 +76,9 @@ class CDSAILabel extends CDSToggleTip {
   @property()
   previousValue;
 
-  connectedCallback() {
-    super.connectedCallback?.();
-    document.addEventListener('click', this._handleOutsideClick, true);
-    document.addEventListener('focusin', this._handleFocusChange, true);
-  }
-  disconnectedCallback() {
-    super.disconnectedCallback?.();
-    document.removeEventListener('click', this._handleOutsideClick, true);
-    document.removeEventListener('click', this._handleFocusChange, true);
-  }
-
+  @HostListener('parentRoot:click')
```

Comment:
> Can you add a temporary test story for reviewers to test these changes?

added, you can test it with changing 
```suggestion
  @HostListener('document:click')
```